modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ViewObjectDefinitionsDisplayContext.java

```diff
@@ -124,6 +116,14 @@ public List<FDSActionDropdownItem> getFDSActionDropdownItems()
 				"get", "permissions", "modal-permissions"));
 	}
 
+	public String getModelBuilderURL() throws Exception {
+		return PortletURLBuilder.create(
+			getPortletURL()
+		).setMVCRenderCommandName(
+			"/object_definitions/edit_object_folder"
+		).buildString();
+	}
+
```

Comment:
Why did you rename but keep using [EditObjectFolder](https://github.com/liferay-objects/liferay-portal/blob/b4a34a493d15f69bd310256feb254303933b340a/modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EditObjectFolder.tsx#L10)?

Reply:
In this case I have the folders that will be edited within the model builder tool. So I tried to connect these two concepts.

I thought about renaming MVC Render and JSP from EditObjectFolder to ModelBuilder. But in BE we don't have the concept of ModelBuilder.

The name of the button that will take us to this page is called "View in Model Builder", I wanted to pass to the button a URL with a name referring to the ModelBuilder.

I really believe that we can connect these two concepts in a better way. Perhaps in the HashMapBuilder calls the modelBuilderURL key and calls the EditObjectFolderURL method. What do you think?

Reply:
I think we should maintain the pattern and name the method relating what it does.
In this case I suggest to keep the method name, and don't change it, since the method is returning the `edit_object_folder` URL.

Reply:
I came up with the name of this method in the previous PR and now I believe I can better name the MVC Render and JSP method to point to the ModelBuilder.

I'm thinking of changing rename the `EditObjectFolderMVCRenderCommand` to `ModelBuilderMVCRenderCommand`, the JSP `edit_object_folder.jsp` to `model_builder.jsp` and consequently the return of this method.

Reply:
Great suggestion, if you rename the others it will fit better.